### PR TITLE
New package: sfxr-1.2.1

### PR DESCRIPTION
**Pull request by agausmann**

### CODE REVIEW

**Comment by agausmann** [comment]
Compiles, but crashes with a segmentation fault when ran.

File: srcpkgs/sfxr/files/LICENSE [hunk 1]
```diff
@@ -0,0 +1,32 @@
+License
```
Comment (abenson) [inline srcpkgs/sfxr/files/LICENSE:1]: Don't vendor the license. instead you can add

```
post_extract() {
	sed -n '123,154p' readme.txt > LICENSE
}
```

and `vlicense` that

File: srcpkgs/sfxr/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+pkgname=sfxr
+version=1.2.1
+revision=1
+wrksrc="sfxr-sdl-${version}"
+build_style=gnu-makefile
+hostmakedepends="pkg-config desktop-file-utils"
+makedepends="SDL-devel gtk+3-devel"
+short_desc="Sound effect generator"
+maintainer="Adam Gausmann <agausmann@fastmail.com>"
+license="custom:sfxr, MIT"
```
Comment (abenson) [inline srcpkgs/sfxr/template:11]: I think just MIT is fine for license, given the comments in the readme.

**Comment by abenson** [comment]
Seemingly abandoned.

**Pull request closed without merging**